subprojects/instrumentation-support/src/main/java/org/gradle/profiler/studio/instrumented/Interceptor.java

```diff
@@ -38,11 +38,11 @@ public static void onConnect(DefaultGradleConnector connector) {
         System.out.println("* Starting tooling API operation " + operation);
         int id = COUNTER.incrementAndGet();
 
-        SyncParameters syncParameters;
+        GradleInvocationParameters syncParameters;
         try {
             Client client = Client.INSTANCE;
-            client.send(new SyncStarted(id));
-            syncParameters = client.receiveSyncParameters(Duration.ofSeconds(60));
+            client.send(new GradleInvocationStarted(id));
+            syncParameters = client.receiveSyncParameters(Duration.ofSeconds(300));
```

Comment:
Another wait time which could be extracted to somewhere.

Reply:
done

Reply:
Also I don't know why I changed that to 300 so I reverted it back to 60.